rules/python/shared/django/http_response.yml

```diff
@@ -0,0 +1,21 @@
+type: shared
+imports:
+  - python_shared_lang_import2
+languages:
+  - python
+patterns:
+  - pattern: $<HTTP_RESPONSE>
+    filters:
+      - variable: HTTP_RESPONSE
+        detection: python_shared_lang_import2
```

Comment:
Maybe this rule can be an instance? Depending on the outcome of [this](https://github.com/Bearer/bearer-rules/pull/393#discussion_r1603001937)